docs/quickstart/index.rst

```diff
@@ -3,39 +3,149 @@
+Start Confluent and KSQL
+========================
 
 To get started, you must start a Kafka cluster, including ZooKeeper and a Kafka broker. KSQL will then query messages from this Kafka cluster.
 
-Launch a Kafka cluster, start KSQL, and produce messages to query. Follow the instructions based on whether you are using Docker:
+-----------
+Start Kafka
+-----------
+
+Navigate to the ``confluent-4.1.0`` directory and start |cp| using the Confluent CLI. ZooKeeper is listening on ``localhost:2181``,
+Kafka broker is listening on ``localhost:9092``, and Confluent Schema Registry is listening on ``localhost:8081``.
+
+.. code:: bash
+
+    $ <path-to-confluent>/bin/confluent start
+
+Your output should resemble this.
+
+.. code:: bash
+
+    Starting zookeeper
+    zookeeper is [UP]
+    Starting kafka
+    kafka is [UP]
+    Starting schema-registry
+    schema-registry is [UP]
+    Starting kafka-rest
+    kafka-rest is [UP]
+    Starting connect
+    connect is [UP]
+    Starting ksql
+    ksql is [UP]
+
+
+----------
+Start KSQL
+----------
 
-:ref:`ksql_quickstart_docker` (recommended)
-    The Docker container starts a Kafka cluster, starts KSQL, and automatically runs a data generator that continuously produces Kafka messages to the Kafka cluster. No additional setup is required.
+#. Lauch the KSQL CLI. The ``local`` argument starts KSQL in :ref:`standalone
+   mode <modes-of-operation>`.
+
+   .. code:: bash
+
+       $ ./bin/ksql-cli local
+
+   After KSQL is started, your terminal should resemble this.
+
+   .. code:: bash
+
+                          ===========================================
+                          =        _  __ _____  ____  _             =
+                          =       | |/ // ____|/ __ \| |            =
+                          =       | ' /| (___ | |  | | |            =
+                          =       |  <  \___ \| |  | | |            =
+                          =       | . \ ____) | |__| | |____        =
+                          =       |_|\_\_____/ \___\_\______|       =
+                          =                                         =
+                          =  Streaming SQL Engine for Apache Kafka® =
+                          ===========================================
+
+        Copyright 2017 Confluent Inc.
+
+        CLI v0.5, Server v0.5 located at http://localhost:8090
+
+        Having trouble? Type 'help' (case-insensitive) for a rundown of how things work!
+
+        ksql>
+
+See the steps below to generate data to the Kafka cluster.
+
+------------------
+Produce topic data
+------------------
+
+Run these steps to produce data to the Kafka topics ``pageviews`` and ``users``.
+
+1. Produce Kafka data to the ``pageviews`` topic using the data
+   generator. The following example continuously generates data with a
+   value in DELIMITED format.
+
+   .. code:: bash
+
+       $ ./bin/ksql-datagen quickstart=pageviews format=delimited topic=pageviews maxInterval=10000
```

Comment:
Let's change this to `maxInterval=500`

Reply:
@hjafarpour any reason whey we don't make this even smaller? i.e, say 100?